test/get-module-specifier-test.js

```diff
@@ -51,90 +51,51 @@ describe('get-module-specifier', function() {
     moduleConfig = getModuleConfig(resolverConfig);
   });
 
-  it('identifies named modules in the root of a collection as the default type for that collection', function() {
```

Comment:
Yes, still valid. I removed it because this is the case we were discussing. I thought I would remove it from the spec until we could support it "correctly". I'm fine restoring it though.